python_modules/libraries/dagster-snowflake-pandas/dagster_snowflake_pandas/snowflake_pandas_type_handler.py

```diff
@@ -21,7 +22,25 @@ def _connect_snowflake(context: Union[InputContext, OutputContext], table_slice:
     ).get_connection(raw_conn=False)
 
 
-class SnowflakePandasTypeHandler(DbTypeHandler[DataFrame]):
+def _convert_timestamp_to_date(s: pd.Series) -> pd.Series:
```

Comment:
This should be called `_convert_pandas_timestamp_to_python_timestamp` (or similar) because we are converting to a `datetime` object, not a plain `date` right? Nor would be want to convert to a `date` in the general case because that would destroy user information (the time).

Reply:
names changed to reflect converting to strings now